Legal/EULA.txt

```diff
@@ -424,3 +425,5 @@ B.9.16 Notice. Customer may send notices required under this Agreement to Esri a
     Tel.: 909-793-2853
     Email: LegalNotices@esri.com
 
+  
+

```

Comment:
Just double checking: Were these additional blank lines deliberate?

Reply:
I believe so. We get this file directly from the legal team